deploy/03 - ApproveTokens.ts

```diff
@@ -18,8 +18,48 @@ const func: DeployFunction = async (hre: HardhatRuntimeEnvironment) => {
     const {deployments, getNamedAccounts, network, ethers} = hre;
 
     if (network.live) {
-      const {dai, usdc, usdt, busd, btcb, eth, xrp, jUSDT, jWBTC, jDAI} = await getNamedAccounts();
-      const tokens = [dai, usdc, usdt, busd, btcb, eth, xrp, jUSDT, jWBTC, jDAI];
+      const {
+        dai,
+        usdc,
+        usdt,
+        busd,
+        btcb,
+        eth,
+        xrp,
+        jUSDT,
+        jWBTC,
+        jDAI,
+        wbnb,
+        wbtc,
+        ada,
+        doge,
+        shiba,
+        link,
+        uni,
+        axs,
+        slp,
+      } = await getNamedAccounts();
+      const tokens = [
+        dai,
+        usdc,
+        usdt,
+        busd,
+        btcb,
+        eth,
+        xrp,
+        jUSDT,
+        jWBTC,
+        jDAI,
+        wbnb,
+        wbtc,
+        ada,
+        doge,
+        shiba,
+        link,
+        uni,
+        axs,
+        slp,
+      ];
       const tokensToApprove = tokens.filter((x) => !!x);
```

Comment:
What is this line?
An array of true?

Reply:
the named account will depende on the chain you are, for example doge will be undefined on Ethereum, and uni will be undefined on BSC.
The idea is to filter to get only the named accounts corresponding to the current blockchain.
At this point this probably needs a refactor as it's getting big, but im not sure in how to iterate through named accounts and diferentiate between tokens and other accounts. Maybe put them on another structure?